src/mlpack/core/optimizers/pso/test_function.hpp

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file test_function.hpp
```

Comment:
You can use the existing Rosenbrock function: ``https://github.com/mlpack/mlpack/tree/master/src/mlpack/core/optimizers/problems``